src/applications/disability-benefits/all-claims/tests/all-claims.cypress.spec.js

```diff
@@ -0,0 +1,107 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../config/form';
+import manifest from '../manifest.json';
+import { mockItf } from './all-claims.cypress.helpers';
+
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+    dataSets: ['maximal-test', 'minimal-test', 'newOnly-test'],
+    fixtures: {
+      data: path.join(__dirname, 'data'),
+      mocks: path.join(__dirname, 'fixtures', 'mocks'),
```

Comment:
Although not absolutely necessary, fixtures can be convenient. As shown here, it can make a whole folder of static data (`fixtures/mocks`) available without importing individual objects before referencing them.